src/app/cases/case_components/case-follow-up-add-visit/case-follow-up-add-visit-data.ts

```diff
@@ -0,0 +1,345 @@
+import {
+  FORM_DATA_RADIO,
+  FORM_DATA_DATE,
+  FORM_DATA_SELECT,
+  FORM_DATA_INPUT,
+  FORM_DATA_WIDGET,
+  YesNoUnknown,
+  TemperatureSource,
+  FORM_DATA_DATETIME,
+  CaseFollowupStatus,
+} from '../../../app.constants';
+
+import { EnumToKeyValuePipe } from '../../../_pipes/enum-to-key-value/enum-to-key-value.pipe';
+
+const pipe = new EnumToKeyValuePipe();
+const range = (start: number, end: number, step = 1) => {
+  const output = [];
+  for (let i = start; i < end; i += step) {
+    output.push(i);
+  }
+  return output;
+};
+
+const optionsYesNoUnknown = pipe.transform(YesNoUnknown);
+const optionsAvailability = pipe.transform(CaseFollowupStatus);
+const optionsTemperatureSource = pipe.transform(TemperatureSource);
+const optionsTemperature = range(35, 44.1, 0.1).map((val) => ({
+  key: Number(val.toFixed(1)),
+  value: `${val.toFixed(1)} °C`,
+}));
+
+export const FORM_DATA_CASE_FOLLOW_UP_NEW_VISITS = [
+  {
+    id: 'details',
+    title: 'visitDeatiles',
```

Comment:
spelling error (in the translation files too)

Reply:
corrected.